From 931cefd0d553d4baea835baa276e2767b46423bb Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Wed, 27 Apr 2016 10:50:50 +0800
Subject: [PATCH] ARM64: dts: rk3399-evb: add mmc-hs400-enhanced-strobe support

This patch enables mmc-hs400-enhanced-strobe for rk3399-evb, so
enhanced strobe function will be used if any eMMC 5.1 is probed.

Change-Id: If5f30e0d759f7a9850bec82c3d53d9bb26ba8c3d
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
index 574482bd2cba..d0549f7d10c9 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
@@ -275,6 +275,7 @@
 	mmc-hs400-1_8v;
 	supports-emmc;
 	non-removable;
+	mmc-hs400-enhanced-strobe;
 	status = "okay";
 };
 
-- 
2.35.3

